### ISSUE TITLE
Traffic recorded even with Record not enabled

### ISSUE THREAD

**Issue by wabiloo** [ProxymanApp/Proxyman#2005]
## Description

I have noticed that if I have the proxy overridden, and scripting enabled which applies to some URLs, those URLs will be modified and recorded *even if* the "Record" button is not activated.

Is that normal?

**Comment by NghiaTranUIT** [comment]
yes, it's normal because the Proxy Overriden is enabled, it means your traffic is still going through Proxyman, which can be modified by the Scripting tool.

The Record Button just toggles whether or not it should be recorded and shows in the Main Table View.

**Comment by wabiloo** [comment]
But that's what I'm saying. It appears in the Main Table View (when modified by scripting) even if I am not recording...

**Comment by NghiaTranUIT** [comment]
so, it's a bug. Let me investigate and fix it 👍

**Comment by NghiaTranUIT** [comment]
@wabiloo I decided to keep this current logic because if we ignore all requests, which is modified by debugging tools, it will conflict with another useful case:

1. Capture a bunch of requests on the main table
2. Turn OFF the Recording button 
3. Set up some Breakpoint, Scripting tool
4. Right-click on certain requests -> Repeat 
5. Expect: New repeated requests should be modified by Breakpoint/Scripting and also displayed on the main table.

It's useful for debugging because we can experiment while not recording any new traffic.
